files/zh-cn/web/javascript/guide/regular_expressions/quantifiers/index.md

```diff
@@ -38,7 +38,7 @@ slug: Web/JavaScript/Guide/Regular_expressions/Quantifiers
       <td>
         <p>
           将前一项“x”匹配 1
-          次或更多次。等价于{1,}。例如，/a+/匹配“candy”中的“a”和“caaaaaaandy”中的“a”。
+          次或更多次。等价于{1,}。例如，/a+/匹配“candy”中的“a”和“caaaaaaandy”中的“aaaaaaa”。
```

Comment:
https://github.com/mdn/content/blob/main/files/en-us/web/javascript/guide/regular_expressions/quantifiers/index.md?plain=1#L44

```suggestion
          次或更多次。等价于 <code>{1,}</code>。例如，<code>/a+/</code> 匹配“candy”中的“a”和“caaaaaaandy”中的所有“a”。
```

Reply:
Hey @qwe1164915372. Do you forget to push the commit for this review? I saw you have marked this as resolved.

Reply:
hello,I just check the branch "patch-1" of my "forked" and it’s what I modified.I want to know where to push, please give me some tips to complete this PR, thank you very much.
![a1](https://github.com/mdn/translated-content/assets/31558384/5d57a6de-1e6d-4926-84c4-a06607017187)

Reply:
Just push commit to your remote branch [patch-1](https://github.com/qwe1164915372/translated-content/tree/patch-1). And all those commits would be included in this PR.